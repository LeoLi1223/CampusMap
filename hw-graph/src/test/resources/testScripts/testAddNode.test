# Tests that a node can be added, one single node can be
# printed and no edge from the node is printed.

# Create a graph
CreateGraph graph1

# Add multiple distinct nodes
AddNode graph1 n1

# Print the node in the graph
ListNodes graph1
ListChildren graph1 n1