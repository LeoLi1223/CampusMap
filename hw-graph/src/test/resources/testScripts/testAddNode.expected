# Tests that a node can be added, one single node can be
# printed and no edge from the node is printed.

# Create a graph
created graph graph1

# Add multiple distinct nodes
added node n1 to graph1

# Print the node in the graph
graph1 contains: n1
the children of n1 in graph1 are: